widget/accordion.go

```diff
@@ -151,13 +158,17 @@ func (r *accordionRenderer) Layout(size fyne.Size) {
 			d.Resize(fyne.NewSize(size.Width, min))
 			y += min
 		}
+
+		if i < len(r.container.Items)-1 {
+			y += theme.Padding()
+		}
 	}
 }
 
 func (r *accordionRenderer) MinSize() (size fyne.Size) {
 	for i, ai := range r.container.Items {
 		if i != 0 {
-			size.Height += theme.Padding()
+			size.Height += theme.Padding()*2 + accordionDividerHeight
```

Comment:
Originally this code only added padding between items. With this change an Accordion with a single item adds no padding - which is correct, but one with two items will have `padding*2+1` - I'm not sure about `*2`, doesn't this put double padding between items, what is the rationale here?

Reply:
Between each item is pad+divide+pad, hence padding*2+1.
It was originally just padding, which resulted in pad/2+div+pad/2 and we agreed it looked squashed.
See the test images if you're unsure and let me know what yout think.